theme/ckeditor5-table/tablecellproperties.css

```diff
@@ -15,6 +16,24 @@
 		&.ck-table-cell-properties-form__alignment-row {
 			& .ck.ck-toolbar {
 				background: none;
+
+				& .ck-button {
+					margin: 0;
+
+					@mixin ck-dir rtl {
```

Comment:
If anything is wrong with the toolbar in RTL, it should be fixed [in the toolbar component (.ck-toolbar_compact)](https://github.com/ckeditor/ckeditor5-theme-lark/blob/i/6107/theme/ckeditor5-ui/components/toolbar/toolbar.css#L62), not here. This way the problem will not re-appear in the future in other features/components/forms.